src/client/packaging/pypi/CHANGELOG.md

```diff
@@ -3,6 +3,38 @@
 All notable future changes to the `delphi_epidata` python client will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
+## [4.1.21] - TBD
+
+### Includes
+- https://github.com/cmu-delphi/delphi-epidata/pull/1418
+- https://github.com/cmu-delphi/delphi-epidata/pull/1436
+
+### Added
+- Adds two debug flags: `debug` logs info about HTTP requests and responses; whereas `sandbox` prevents any HTTP requests from actually executing, allowing for tests that do not incur server load.
+
+### Changed
+- Updates internal version detection to use `bumpversion`.
```

Comment:
Lets keep this PR focused on what is already committed; #1436 should make its own updates to the changelog.
```suggestion

### Added
- Adds two debug flags:
  - `debug` logs info about HTTP requests and responses
  - `sandbox` prevents any HTTP requests from actually executing, allowing for tests that do not incur server load.
```

Reply:
you took out the change from #1418!

Reply:
added this in again!